### PR TITLE
Fx usage updates

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: jobs/fx_usage_report.sh [hunk 1]
```diff
@@ -12,4 +12,5 @@ cd Fx_Usage_Report
 python usage_report/usage_report.py \
     --date $date \
     --sample 10 \
-    --output-bucket $bucket
+    --output-bucket $bucket \
+    --output-prefix "$deploy_environment/usage_report_data"
```
Comment (sunahsuh) [inline jobs/fx_usage_report.sh]: The intent here is to make it easier to figure out what's writing the data, I'm guessing?

File: dags/fx_usage_report.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
     task_id='wait_for_main_summary',
     external_dag_id='main_summary',
     external_task_id='main_summary',
-    execution_delta=timedelta(hours = -1), # main_summary waits one hour
+    execution_delta=timedelta(days=-7, hours=-1), # main_summary waits one hour, execution date is beginning of the week
```
Comment (sunahsuh) [inline dags/fx_usage_report.py]: Ugh, airflow.

**Comment by fbertsch** [comment]
@sunahsuh yes it is intentional, but unfortunately Airflow doesn't actually allow me to backfill those dates. I'll leave it in because I'm backfilling manually

File: dags/fx_usage_report.py (hunk 2 above)
Comment (fbertsch) [reply dags/fx_usage_report.py]: yeah, great stuff :)

File: jobs/fx_usage_report.sh (hunk 1 above)
Comment (fbertsch) [reply jobs/fx_usage_report.sh]: Yeah, eventually I'll move to have code first deployed to stage and then to prod. So for now these will both be the same.

**Pull request merged**